lib/organization/Widgets/mydrawer.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+class mydrawer extends StatefulWidget {
+  const mydrawer({Key? key}) : super(key: key);
+
+  @override
+  State<mydrawer> createState() => _mydrawerState();
+}
+
+class _mydrawerState extends State<mydrawer> {
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+    backgroundColor: Color.fromARGB(255, 166, 223, 250),
```

Comment:
Please change the background color to the one shown in the design files. And use RGBO notation as it will help in maintaining uniformity.